mimic/model/customer_objects.py

```diff
@@ -0,0 +1,84 @@
+"""
+Customet Contacts storage object
+"""
+import time
+from characteristic import attributes, Attribute
+
+
+@attributes(["tenant_id", "email_address", "role",
+             Attribute("firstName", default_value="Test FirstName"),
+             Attribute("lastName", default_value="Test LastName")])
+class Contact(object):
+    """
+    A :obj:`Contact` is a representation for each contact for a tenant.
+    """
+
+    static_defaults = {
+        "firstName": "Pat",
+        "lastName": "Example",
+        # "customerAccountNumber": req.params.tenantId,
+        "customerAccountType": "CLOUD",
+        "contactNumber": "RPN-994-758-811",
+        "rcn": "RCN-348-367-072",
+        "emailAddresses": {
+                    "emailAddress": [
+                        {
+                            "primary": True,
+                            "address": None
+                        }
+                    ]
+        },
+        "phoneNumbers": {
+            "phoneNumber": [
+                        {
+                            "number": "55555555555"
+                        }
+            ]
+        },
+        "roles": {
+            "role": []
+        },
+        "link": [
+            {
+                "rel": "via",
+                "href": "http://link-to-nothing-for-contact-in-customer-api"
+            }
+        ]
+    }
+
+    def generate_contacts(self, tenant_id):
```

Comment:
You use this as a class method in the code below, but it's not annotated to be a class method here.  This will cause run-time errors if not addressed.